2021-06-16 01:35:07,374 INFO    MainThread:581734 [wandb_setup.py:_flush():69] setting env: {}
2021-06-16 01:35:07,374 INFO    MainThread:581734 [wandb_setup.py:_flush():69] setting login settings: {}
2021-06-16 01:35:07,375 INFO    MainThread:581734 [wandb_init.py:_log_setup():337] Logging user logs to /home/chengzilong/DDQN/wandb/run-20210616_013507-2xyfwkrz/logs/debug.log
2021-06-16 01:35:07,375 INFO    MainThread:581734 [wandb_init.py:_log_setup():338] Logging internal logs to /home/chengzilong/DDQN/wandb/run-20210616_013507-2xyfwkrz/logs/debug-internal.log
2021-06-16 01:35:07,375 INFO    MainThread:581734 [wandb_init.py:init():370] calling init triggers
2021-06-16 01:35:07,375 INFO    MainThread:581734 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {'gamma': 0.99, 'lr': 0.00025, 'opt_eps': 0.0003125, 'gradient_clip': 0.5, 'batch_size': 32, 'seed': 5, 'eps_start': 1, 'eps_end': 0.01, 'eps_decay_steps': 1000000, 'buffer_size': 1000000, 'env': 'BreakoutNoFrameskip-v4', 'total_steps': 10000000, 'start_updating': 50000, 'train_freq': 4, 'update_target_freq': 10000, 'num_atoms': 51, 'v_min': -10.0, 'v_max': 10.0}
2021-06-16 01:35:07,375 INFO    MainThread:581734 [wandb_init.py:init():419] starting backend
2021-06-16 01:35:07,375 INFO    MainThread:581734 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-06-16 01:35:07,390 INFO    MainThread:581734 [backend.py:ensure_launched():135] starting backend process...
2021-06-16 01:35:07,399 INFO    MainThread:581734 [backend.py:ensure_launched():140] started backend process with pid: 581913
2021-06-16 01:35:07,401 INFO    MainThread:581734 [wandb_init.py:init():424] backend started and connected
2021-06-16 01:35:07,403 INFO    MainThread:581734 [wandb_init.py:init():472] updated telemetry
2021-06-16 01:35:07,404 INFO    MainThread:581734 [wandb_init.py:init():491] communicating current version
2021-06-16 01:35:08,127 INFO    MainThread:581734 [wandb_init.py:init():496] got version response 
2021-06-16 01:35:08,128 INFO    MainThread:581734 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-06-16 01:35:08,399 INFO    MainThread:581734 [wandb_init.py:init():529] starting run threads in backend
2021-06-16 01:35:08,637 WARNING MainThread:581734 [wandb_init.py:init():777] interrupted
Traceback (most recent call last):
  File "/home/chengzilong/anaconda3/envs/myenv/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 757, in init
    run = wi.init()
  File "/home/chengzilong/anaconda3/envs/myenv/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 532, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/home/chengzilong/anaconda3/envs/myenv/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 807, in communicate_run_start
    result = self._communicate(rec)
  File "/home/chengzilong/anaconda3/envs/myenv/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 545, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/chengzilong/anaconda3/envs/myenv/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 82, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/chengzilong/anaconda3/envs/myenv/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/home/chengzilong/anaconda3/envs/myenv/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
